modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
+
+To run the upgrade:
+
+. Using your terminal open on a controller node, switch to the root user. Paste the command you copied in step 5 of upgrading the Runtime Fabric appliance: 
 +
 ----
 sudo /opt/anypoint/runtimefabric/rtfctl appliance upgrade --url <cluster-installer-url>
 ----
 +
-This command also supports using the `--file` parameter to reference an installer package already downloaded on the node.
-+
 A confirmation is displayed that indicates the upgrade is being performed in the background.
-+
-.. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
-.. Monitor the upgrade procedure progress:
-... Using your terminal on a controller node, run the following command: 
+. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
+. To Monitor the upgrade procedure progress, using your terminal on a controller node, run the following command:
 +
 ----
 sudo /opt/anypoint/runtimefabric/installer/gravity plan
 ----
-+
-The upgrade can take more than an hour to complete.
-+
-The connection between Runtime Fabric and Runtime Manager can disconnect multiple times during the upgrade procedure. Connectivity is restored after the upgrade finishes.
-+
-The upgrade procedure is designed to keep deployed applications running and available to serve requests. The upgrade is applied in a rolling method:
-+
-.... A node is selected by the procedure to apply the upgrade.
-.... Applications running on the selected node are redeployed to another node.
-.... The selected node is removed from the cluster.
-.... The upgrade is applied on the selected node.
-.... The selected node is added to the cluster after upgrading.
-
-.. To confirm that the upgrade has completed successfully run the following command:
+. To confirm that the upgrade has completed successfully run the following command:
 +
 ----
 sudo gravity status
 ----
-+
-Then, verify that the cluster status transitioned from *Updating* to *Active*. 
 
+. Verify that the cluster status transitioned from *Updating* to *Active*. 
 . If the Runtime Fabric cluster version was 1.0.x prior to upgrading and an HTTP proxy is in use, run the following command to apply the HTTP proxy settings:
 +
 ----
 sudo ./rtfctl apply http-proxy --confirm existing
 ----
 
-. Perform the following step on *every node* to ensure that system configurations are current:
+=== Upgrade Node Configurations
 
-.. Open a terminal to a Runtime Fabric controller or worker node.
-.. If needed, download the latest `rtfctl` command-line utility:
-+
-----
-cd /opt/anypoint/runtimefabric
-curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
-----
-+
-.. Change file permissions for the `rtfctl` binary: 
-+
-----
-chmod +x rtfctl
-----
-+
-.. Run the `apply system-configurations` command in `rtfctl`:
+Perform the following step on *every node* to ensure that system configurations are current:
```

Comment:
When formatting for emphasis (not recommended), use italic (_ _) rather than bold (* *).